test/templates/panel-grid/index.njk

```diff
@@ -1,23 +1,17 @@
 {% extends "layout.njk" %}
 {% from "./../format.njk" import format %}
 {% from "./component.njk" import panel_grid %}
-{% from "./../panel/component.njk" import panel, panel_header, panel_actions, panel_filters, panel_body, panel_footer %}
 
 {% set panel_item %}
-{% call panel() -%}
-    {% call panel_body() -%}
-      <p>.fd-panel</p>
-    {%- endcall %}
-{%- endcall %}
+    <div class="panel-playground">.fd-panel</div>
```

Comment:
Can we change `.fd-panel` here to `panel-playground` or `placeholder` so that we know this is a placeholder element?